frontend/packages/dev-console/integration-tests/OWNERS

```diff
@@ -0,0 +1,23 @@
+reviewers:
+  - abhi-kn
+  - andrewballantyne
+  - abhinandan13jan
+  - debsmita1
+  - divyanshiGupta
+  - christianvogt
+  - invincibleJai
+  - jerolimov
+  - karthikjeeyar
+  - nemesis09
+  - rohitkrai03
+  - rottencandy
+  - sahil143
+  - vikram-raj
+approvers:
+  - andrewballantyne
+  - christianvogt
+  - invincibleJai
+  - rohitkrai03
+  - makambalaji
```

Comment:
You don't have to define everyone... just what you want extra on top of the OWNER files above. It's a stacking list.

Reply:
Example of another one I did for Jeff so he could review the stuff he has been working on: https://github.com/openshift/console/pull/4471/files

Reply:
@bmakam OWNERS file are additive, simply add yourself as an approver and any QE as reviewers

Reply:
updated the file by only adding reviewers as QE members and approver as my name in the list